apps/projects/manifest.json

```
@@ -1,6 +1,6 @@
 {
   "name": "Projects",
-  "description": "Allocate funding to multiple Github issues in a single action and collectively curate issues.",
+  "description": "Fund bounties and prioritize Github issues.",
```

The correct form is GitHub

```suggestion
  "description": "Fund bounties and prioritize GitHub issues.",
```